training/spam.md

```diff
@@ -31,16 +32,17 @@ However, this is not:
 > To fix your problem, you should foo your bar and pass your baz to quux.  
 > I wrote about this in more detail on my blog (link).
 
-That post discloses the fact that the blog is the author's, so it's okay. This isn't spam
-either:
+That post discloses the fact that the blog is the author's, so it's okay. 
+
+This isn't spam either:
 
 > delete me delete me delete me delete me delete me
 
-If you come across something along those lines, it's most likely self-vandalism - where a
+If you come across something along those lines, it's most likely self-vandalism, which is where a
 post's author has removed a post's content in an effort to delete it. That's not something
 Stack Exchange allows, but it's not spam.
 
-As always, use your best judgement, and ask if you're not sure.
+As always, use your best judgment, and ask if you're not sure.

```

Comment:
Same here.

Reply:
https://english.stackexchange.com/questions/1623/is-the-proper-spelling-judgment-or-judgement

Judgment is more common, even in British English. I left your "organisation" alone :)